doc/source/user-guide/options.rst

```diff
@@ -111,41 +111,50 @@ If you want to hide all icons, use the ``show_icons`` Boolean variable:
 Static search options
 ----------------------
 
-The Ansys Sphinx theme supports static search options to customize the search experience.
+The Ansys Sphinx theme supports static search options to customize the search
+experience.
 
-The static search bar is created using ``Fuse.js``. You can provide all options supported by ``Fuse.js`` through the ``static_search`` dictionary in the ``html_theme_options``.
+The static search bar is created using ``Fuse.js``. You can provide `all
+options <https://www.fusejs.io/api/options.html>`_ supported by ``Fuse.js``
+through the ``static_search`` dictionary in the ``html_theme_options``.
 
-Additional options include:
-
-1. ``keys``: List of keys to search in the documents. Default are ``["title", "text"]``.
-2. ``threshold``: The minimum score a search result must have to be included in the results. Default is ``0.5``.
-3. ``ignoreLocation``: Whether to ignore the location of the search term in the document. Default is ``False``. Ignoring the location can increase the search speed for large documents.
-4. ``limit``: The maximum number of search results to display. Default is ``10``.
-5. ``min_chars_for_search``: The minimum number of characters required to start the search. Default is ``1``.
+To improve the search performance, a debounce function is available. By
+default, a delay of 300 milliseconds is applied. To modify this value, declare
```

Comment:
```suggestion
default, a delay of 150 milliseconds is applied. To modify this value, declare
```
isnt it 150 in the code ?https://github.com/ansys/ansys-sphinx-theme/pull/525/files#diff-ffed2084dacf113e19db068e740252b0240862c86bf3ad1c15a7973c0e01314fR168

Reply:
You are right, fixing the default value to 300ms.